sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -10653,6 +10654,88 @@
               "hideLabel": false,
               "errorLabel": "Disability Information"
             },
+            {
+              "label": "HTML",
+              "labelWidth": "",
+              "labelMargin": "",
+              "tag": "p",
+              "className": "alert alert-warning fa fa-exclamation-triangle w-100",
+              "attrs": [
+                {
+                  "attr": "",
+                  "value": ""
+                }
+              ],
+              "content": "<strong> Your account status indicates you have been approved for Disability status.</strong>\n <br /> If you answer no to this question you will not be assessed or receive disability funding.",
+              "refreshOnChange": false,
+              "customClass": "banner-warning",
+              "hidden": false,
+              "modalEdit": false,
+              "key": "html8",
+              "tags": [],
+              "properties": {},
+              "conditional": {
+                "show": "true",
+                "when": "bcResident",
+                "eq": "no",
+                "json": ""
+              },
+              "customConditional": "show = (!data.readOnly && (data.applicationPDPPDStatus == 'noIDoNotHaveADisability' || data.applicationPDPPDStatus == 'noIWantToAccessOtherFundingTypes') && (data.disabilityStatusValue == 'PD' || data.disabilityStatusValue == 'PPD'));",
+              "logic": [],
+              "attributes": {},
+              "overlay": {
+                "style": "",
+                "page": "",
+                "left": "",
+                "top": "",
+                "width": "",
+                "height": ""
+              },
+              "type": "htmlelement",
+              "input": false,
+              "tableView": false,
+              "placeholder": "",
+              "prefix": "",
+              "suffix": "",
+              "multiple": false,
+              "defaultValue": null,
+              "protected": false,
+              "unique": false,
+              "persistent": false,
```

Comment:
This variable doesn't have to be persistent as it doesn't need to be stored anywhere.

![image](https://github.com/bcgov/SIMS/assets/54600590/8a69c59c-8101-4949-a5be-ccff3a7d4dfb)

Reply:
Thanks, this should now be updated on all the forms.